modules/pruning-images.adoc

```diff
@@ -50,23 +52,59 @@ status:
     reason: Succeeded
     message: "Most recent image pruning job succeeded."
 ----
-<1> `schedule`: `CronJob` formatted schedule. This is an optional field, default is daily at midnight.
-<2> `suspend`: If set to `true`, the `CronJob` running pruning is suspended. This is an optional field, default is `false`. The initial value on new clusters is `false`.
-<3> `keepTagRevisions`: The number of revisions per tag to keep. This is an optional field, default is `3`. The initial value is `3`.
-<4> `keepYoungerThanDuration`: Retain images younger than this duration. This is an optional field. If a value is not specified, either `keepYoungerThan` or the default value `60m` (60 minutes) is used.
-<5> `keepYoungerThan`: Deprecated. The same as `keepYoungerThanDuration`, but the duration is specified as an integer in nanoseconds. This is an optional field. When `keepYoungerThanDuration` is set, this field is ignored.
-<6> `resources`: Standard Pod resource requests and limits. This is an optional field.
-<7> `affinity`: Standard Pod affinity. This is an optional field.
-<8> `nodeSelector`: Standard Pod node selector. This is an optional field.
-<9> `tolerations`: Standard Pod tolerations. This is an optional field.
-<10> `successfulJobsHistoryLimit`: The maximum number of successful jobs to retain. Must be `>= 1` to ensure metrics are reported. This is an optional field, default is `3`. The initial value is `3`.
-<11> `failedJobsHistoryLimit`: The maximum number of failed jobs to retain. Must be `>= 1` to ensure metrics are reported. This is an optional field, default is `3`. The initial value is `3`.
-<12> `observedGeneration`: The generation observed by the Operator.
-<13> `conditions`: The standard condition objects with the following types:
-* `Available`: Indicates if the pruning job has been created. Reasons can be Ready or Error.
-* `Scheduled`: Indicates if the next pruning job has been scheduled. Reasons can be Scheduled, Suspended, or Error.
-* `Failed`: Indicates if the most recent pruning job failed.
+<1> Optional. `CronJob` formatted `schedule`. The default setting is daily at midnight.
+<2> Optional. If `suspend` is set to `true`, the `CronJob` running pruning is suspended. The `suspend` field is set to `false` by default. The initial value on new clusters is `false`.
+<3> Optional. `keepTagRevisions` is the number of revisions per tag to keep. The  default value is `3`. The initial value is `3`.
```

Comment:
@bmcelvee We may need add logLevel here like other spec parameters
logLevel sets the level of log output for the pruner job. Valid values are: "Normal", "Debug", "Trace", "TraceAll". Defaults to "Normal".



logLevel sets the level of log output for the pruner job. Valid values are: "Normal", "Debug", "Trace", "TraceAll". Defaults to "Normal".
--
 

<br class="Apple-interchange-newline">